garde_derive/src/syntax.rs

```diff
@@ -271,6 +271,41 @@ impl Parse for model::RawRule {
             };
         }
 
+        macro_rules! error_if_missing_feature {
+            ($rule:expr, $feature:expr) => {
+                #[cfg(not(feature = $feature))]
+                proc_macro_error::abort!(
```

Comment:
We shouldn't immediately abort when we find a missing feature. The other rules may be enabled and should still be parsed and checked. It's fine if each instance of a rule reports the same "missing feature" error.

Reply:
Thanks! 
It looks like using `syn::Error` produces the same output as `abort!()`.
I switched to `syn::Error` and updated the PR to drop adding the `proc_macro_error` crate.

Reply:
I kept the 
```
error: Validation rule `{rule}` not found. Did you forget to add the `{flag}` feature flag?
```

error message format because I think it is a bit nicer of an error message for the crate user. 

But I can change to a more generic `missing feature` error if you think that is better.